src/CloudSeis.jl

```diff
@@ -1134,9 +1154,12 @@ close(io)
 TeaSeis.set!(prop::TraceProperty, hdrs::AbstractArray{UInt8,2}, i::Integer, value) = set!(prop, @view(hdrs[:,i]), value)
 
 Base.LinearIndices(io::CSeis) = LinearIndices(size(io)[3:end])
+linearframeidx(io, idx::NTuple{N,Int}) where {N} = LinearIndices(io)[CartesianIndex(ntuple(idim->div(idx[idim] - io.axis_lstarts[2+idim], io.axis_lincs[2+idim]) + 1, N))]
```

Comment:
@samtkaplan does this handle logical indices that are not valid? i.e. does it produce an error if you try converting logical index 2 to a linear index, when the logical start is 1 and the increment is 2?

Reply:
I think that in this case `2` would convert to `1`, I think.  So, it might be nice to have more error checking here, perhaps at the expense of performance.  Thanks for the note!

Reply:
@mifkovic-chevron added the check.  not sure if it matters, but the check can be elided via the `@inbounds` macro.

Reply:
Looks good! I think most people familiar with the `@inbounds` macro are probably aware that they're disabling a safety check. So I don't think we need to worry about that.